python/openmldb_tool/tests/log_parser_test.py

```diff
@@ -0,0 +1,20 @@
+import os
+import re
+
+import pytest
+from diagnostic_tool.parser import log_parser
+
+err_log_list = os.listdir("off_err_logs")
+
+
+@pytest.fixture(params=err_log_list)
+def log_fixture(request):
```

Comment:
Is it a little complex? It's simpler to just use `@pytest.mark.parametrize(['file'], file-list-in-logs-dir)` or `@pytest.fixture
def log_fixture(): [ 'off_err_logs' + file_name for file_name  in os.listdir("off_err_logs")`